qutebrowser/completion/models/miscmodels.py

```diff
@@ -32,6 +33,8 @@ class CommandCompletionModel(base.BaseCompletionModel):
 
     """A CompletionModel filled with all commands and descriptions."""
 
+    COLUMN_WIDTHS = (20, 60, 20)
+
```

Comment:
Huh, looks like that [broke the pylint comment](https://travis-ci.org/The-Compiler/qutebrowser/jobs/131005701) - can you try moving it after the comment?